ror-tools/src/test/scala/tech/beshu/ror/tools/utils/TestEsContainerManager.scala

```diff
@@ -0,0 +1,63 @@
+/*
+ *    This file is part of ReadonlyREST.
+ *
+ *    ReadonlyREST is free software: you can redistribute it and/or modify
+ *    it under the terms of the GNU General Public License as published by
+ *    the Free Software Foundation, either version 3 of the License, or
+ *    (at your option) any later version.
+ *
+ *    ReadonlyREST is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *    GNU General Public License for more details.
+ *
+ *    You should have received a copy of the GNU General Public License
+ *    along with ReadonlyREST.  If not, see http://www.gnu.org/licenses/
+ */
+package tech.beshu.ror.tools.utils
+
+import cats.data.NonEmptyList
+import monix.eval.Task
+import monix.execution.atomic.AtomicInt
+import tech.beshu.ror.utils.containers.*
+import tech.beshu.ror.utils.containers.EsContainerCreator.EsNodeSettings
+import tech.beshu.ror.utils.containers.images.ReadonlyRestWithEnabledXpackSecurityPlugin
+import tech.beshu.ror.utils.containers.images.domain.Enabled
+
+import java.io.Console as _
+import scala.concurrent.duration.*
+import scala.language.postfixOps
+
+class TestEsContainerManager extends EsContainerCreator {
```

Comment:
I'm not sure about the name. I don't like the Manager suffix for sure. Looking at the implementation (and its usage in tests), I'd say it's an example ES container with ROR installed. So maybe `ExampleEsWithRorContainer`? WDYT?

Reply:
Renamed and added comment about what is happening in this container and how/why it is different.